api/CloseEvent.json

```diff
@@ -116,10 +116,10 @@
             },
             "oculus": "mirror",
             "opera": {
-              "version_added": "≤15"
+              "version_added": "12.1"
```

Comment:
No range? Did you test two versions of Opera?

Reply:
The collector did generate a range, but `CloseEvent` itself is set to `12.1`!

Reply:
Aha, that makes sense!